rancher2/schema_cluster_rke_config_cloud_provider_vsphere.go

```diff
@@ -51,6 +51,11 @@ func clusterRKEConfigCloudProviderVsphereGlobalFields() map[string]*schema.Schem
 			Optional: true,
 			Computed: true,
 		},
+		"graceful_shutdown_timeout": {
```

Comment:
Add a `Description` to this field and reference it in the `rancher2_cluster` docs

Reply:
No entry in this list has a description, so I guess it is the same as this one. 
The description is added in another place; docs are updated too.